internal/impl/devbox.go

```diff
@@ -653,6 +685,63 @@ func (d *Devbox) installNixProfile() (err error) {
 	return
 }
 
+// writeScriptsToFiles writes scripts defined in devbox.json into files inside .devbox/gen/scripts.
+// Scripts (and hooks) are persisted so that we can easily call them from devbox run (inside or outside shell).
+func (d *Devbox) writeScriptsToFiles() error {
+	err := os.MkdirAll(filepath.Join(d.projectDir, scriptsDir), 0755) // Ensure directory exists.
+	if err != nil {
+		return err
```

Comment:
errors.WithStack?

Reply:
sure, i'll add, but.. does anyone ever use that stack anyway?  I question its usefulness :)